bootstrap/bootstrap

```diff
@@ -794,7 +794,7 @@ SunOS)
 	else
 		need_sed=yes
 	fi
-	if [ "`uname -r`" = "5.11" -a -x "/usr/bin/bash" ]; then
+	if [ "`uname -r`" = "5.12" -a -x "/usr/bin/bash" ]; then
```

Comment:
Currently this will not match *any* SunOS system. If it's a mistake then it should be corrected. If it's your intention to not use this clause at all, then it should be entirely removed. Otherwise this is just kicking the can down the road and will cause problems whenever a system identified as SunOS 5.12 is released by somebody.